modules/ROOT/examples/csharp/Program.cs

```diff
@@ -946,4 +962,27 @@ public async Task Send(Message message)
         }
         // end::passive-peer-send[]
     }
+
+    // tag::custom-logging[]
+    private class LogTestLogger : ILogger
+    {
+        [NotNull]
+        private readonly List<string> _lines = new List<string>();
+
+        public IReadOnlyList<string> Lines => _lines;

```

Comment:
@Sandychuang8 would it be possible to remove references to `lines` to match the Swift version https://github.com/couchbase/docs-couchbase-lite/pull/71/files#diff-dc21fc8be6ee8786054080415959e410R728 ? I believe `lines` is used in unit tests.